src/System.Management.Automation/engine/COM/ComDispatch.cs

```diff
@@ -34,7 +36,7 @@ internal interface IDispatch
             int lcid,
             COM.INVOKEKIND wFlags,
             [In, Out][MarshalAs(UnmanagedType.LPArray)] COM.DISPPARAMS[] paramArray,
-            out object pVarResult,
+            out object? pVarResult,
             out ComInvoker.EXCEPINFO pExcepInfo,
```

Comment:
`EXCEPINFO* pExcepInfo`

> Pointer to a structure that contains exception information. This structure should be filled in if DISP_E_EXCEPTION is returned. Can be NULL.

This can be null, but as we use a struct for ComInvoker.EXCEPTINFO, it shouldn't be marked as `?`